src/Application/User/Command/DeleteUserOrganizationCommand.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\User\Command;
+
+use App\Application\CommandInterface;
+use App\Domain\User\OrganizationUser;
+
+final class DeleteUserOrganizationCommand implements CommandInterface
```

Comment:
C'est l'inverse non?

```suggestion
final class DeleteOrganizationUserCommand implements CommandInterface
```